internal/k3d/create.go

```diff
@@ -18,16 +20,23 @@ func CreateK3dCluster() error {
 	// I tried Terraform templates using: https://registry.terraform.io/providers/pvotal-tech/k3d/latest/docs/resources/cluster
 	// it didn't worked as expected.
 	if !viper.GetBool("k3d.created") {
+		volumeFolder := fmt.Sprintf("%s/minio-storage", config.K1FolderPath)
+		err := os.Mkdir(volumeFolder, 0700)
+		if err != nil {
+			log.Info().Err(err)
```

Comment:
it sounds like `log.Error().Err(err).Msg("")` , the current line won't send log since it's missing the `.Msg("")`

feel free to mark as resolved